etna/libs/ts2vec/ts2vec.py

```diff
@@ -95,16 +95,18 @@ def fit(self, train_data, n_epochs=None, n_iters=None, verbose=False):
 
         loss_log = []
 
+        cur_epoch = 0
+        cur_iter = 0
         while True:
```

Comment:
Do I understand correctly that calling `fit` the second time takes weights fitted after the first time and keeps updating them? Is it continuing the learning of the same model?

Reply:
Yes, of course

Reply:
We should clarify this behavior in the docs.